Readme.md

```diff
@@ -10,10 +10,29 @@ Radeon Performance Primitives (RPP) library is a comprehensive high-performance
 
 RPP is developed for __Linux__ operating system.
 
-##### Prerequisites
-1. Ubuntu `16.04`/`18.04`
-2. [ROCm supported hardware](https://rocm.github.io/hardware.html)
-3. [ROCm](https://github.com/RadeonOpenCompute/ROCm#installing-from-amd-rocm-repositories)
+## Prerequisites
+* Ubuntu `16.04`/`18.04`
```

Comment:
Should we say this because there are build errors in 16.04?

Reply:
we have noted the bug, it might be the pre-req issue.